core/Makefile

```diff
@@ -5,6 +5,7 @@ CC:=$(CI_GCC)
 endif
 
 LOADABLE_CFLAGS=-std=c99 -fPIC -shared -Wall
+LIB_CFLAGS=-std=c99 -fPIC -c -Wall
```

Comment:
I don't think we need `-fPIC` for the static library. Removing should reduce the binary size.